docker-compose.yaml

```diff
@@ -46,7 +46,11 @@ services:
       - market-ledger-sync
       - rethink
       - validator
-    command: echo "No REST API command set"
+    command: |
+      bash -c "
+        sleep 3 &&
```

Comment:
The ideal way to handle this is write it so it gracefully attempts to reconnect if the db or validator are unavailable. I will see if I can add this to a later PR for both ledger-sync and the rest-api.